src/acme/ui/AcmeUI.java

```diff
@@ -0,0 +1,151 @@
+    public void importCustomers() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void customerList() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void customerAddUpdate(Customer customer) {
+        // This will be called with null to create
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void userList() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void userAddUpdate(User customer) {
+        // This will be called with null to create
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void importCouriers() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void courierList() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void courierAddUpdate(Courier courier) {
+        // This will be called with null to create
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void mapView() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    // Everyone will tie in their panel like this.  Replace my example with your code
+    public void reports() {
+        ExampleJPanel exampleJPanel = new ExampleJPanel();
+        this.setPanel(exampleJPanel);
+    }
+
+    public Company getCompany() {
+        return company;
+    }
+}
+
+class ShutdownListener implements WindowListener {
+    public void windowClosing(WindowEvent event) {
+
+    }
+    public void windowOpened(WindowEvent event) {}
+    public void windowClosed(WindowEvent event) {}
+    public void windowIconified(WindowEvent event) {}
+    public void windowDeiconified(WindowEvent event) {}
+    public void windowActivated(WindowEvent event) {}
+    public void windowDeactivated(WindowEvent event) {}
+}
```

Comment:
This'll be helpful, thanks for including this!

Reply:
It's required when implementing the interface